crates/sui-indexer-alt-jsonrpc/src/data/objects.rs

```diff
@@ -160,7 +160,7 @@ pub(crate) async fn load_live(
 
     // If the latest object info record has no owner, the object is not live (it is wrapped or
     // deleted).
-    if obj_info.owner_id.is_none() {
+    if obj_info.owner_kind.is_none() {
```

Comment:
This was pretty hard to find -- turns out `owner_id` is None when the object is frozen or shared!